server/README.txt

```diff
@@ -24,6 +24,15 @@ Getting Started
 
     env/bin/pytest
 
+- Copy development.ini.sample to development.ini and fill credentails before first deployment:
```

Comment:
I'm not sure, but may be better to write this instructions into the main README in the Event.me folder

Reply:
I thought, I wrote it into the main README